src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -128,13 +128,21 @@ namespace InternalHeatGains {
 
     // Data
     // MODULE PARAMETER DEFINITIONS:
-    int constexpr ITEClassNone(0);
-    int constexpr ITEClassA1(1);
-    int constexpr ITEClassA2(2);
-    int constexpr ITEClassA3(3);
-    int constexpr ITEClassA4(4);
-    int constexpr ITEClassB(5);
-    int constexpr ITEClassC(6);
+    enum class ITEClass
+    {
+        Invalid = -1,
+        None, // (0)
+        A1,   // (1)
+        A2,   // (2)
+        A3,   // (3)
+        A4,   // (4)
+        B,    // (5)
+        C,    // (6)
+        H1,   // (7)
+        Num
+    };
+    static constexpr std::array<std::string_view, static_cast<int>(ITEClass::Num)> ITEClassNamesUC = {"NONE", "A1", "A2", "A3", "A4", "B", "C", "H1"};
+
     int constexpr ITEInletAdjustedSupply(0);
```

Comment:
These can also be an `enum class` actually.

Reply:
This is now converted to `enum` class as well.